doc/controllers/assets/search/index.md

```diff
@@ -0,0 +1,98 @@
+--- 
+code: true
+type: page
+title: search
+--- 
+
+# search
+
+Searches for assets corresponding to the specified expression and options
+
+--- 
+
+## Query Syntax 
+
+### HTTP 
+
+```http
+URL: http://kuzzle:7512/_plugin/cloudinary/assets/search
+Method: POST
+```
```

Comment:
You should include an example of the required body like this: https://docs.kuzzle.io/core/1/api/controllers/document/search/
This apply for every method with a body